packages/babel-generator/test/index.js

```diff
@@ -16,6 +16,11 @@ suite("generation", function () {
       assert.ok(t.VISITOR_KEYS[type], type + " should not exist");
     });
   });
+
+  test("valid code", function() {
+    // Should not generate `0.foo`
+    new Function(generate.default(t.memberExpression(t.numericLiteral(60702), t.identifier("foo"))).code);
+  });
```

Comment:
haha `new Function`?

Not a big deal but maybe we could separate out the `t.memberExpression(t.numericLiteral(60702), t.identifier("foo"))` as a var?